cg/store/models.py

```diff
@@ -975,6 +976,18 @@ class RunDevice(Base):
     }
 
 
+class IlluminaFlowCellDevice(RunDevice):
+    """Model for storing Illumina flow cells."""
+
+    __tablename__ = "illumina_flow_cell"
+
+    id: Mapped[int] = mapped_column(ForeignKey("run_device.id"), primary_key=True)
+    flow_cell_type: Mapped[IlluminaFlowCellType | None]
+    flow_cell_name: Mapped[UniqueStr64 | None]
```

Comment:
Is this not in the parent class? (RunDevie.internal_id)

Reply:
I am referring here to the full name, so `20231108_LH00188_0028_B22F52TLT3` for example. Is this something relevant in this class or in the metrics class?

Reply:
We don't want to store this string. It is a concatenation of:

date_sequencername_???_flow_cell_name